### PR TITLE
Meetings 

### PR DESCRIPTION
**Pull request by Jdww**

### CODE REVIEW

**Review by tiagobastos — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Comment by tiagobastos** [comment]
✔️ Manually checked

**Review by gdegirmenci — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -3315,6 +3338,143 @@ Get information about a meeting.
...
+                        + area_level_two_id: `db232cf8-ad4a-024b-941f-15a7a74f0fd2`
+                + Properties
+                    + type: `customLocation` (string, required)
+                    + id: `4191c526-5e26-0818-8e16-9523215b5081` (string, required)
+        + milestone_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
+        + deal_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
+        + custom_fields (array[CustomFieldValue], optional)
+
++ Response 201
+
```
Comment (mrtus) [inline apiary.apib]: Q: Response returns an id of the event does it not?

We should probably document that too.

**Review by mrtus — DISMISSED** [review]

File: src/04-calendar/meetings.apib [hunk 2]
```diff
@@ -165,4 +187,146 @@ Get information about a meeting.
...
+                    + id: `4191c526-5e26-0818-8e16-9523215b5081` (string, required)
+        + milestone_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
+        + deal_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
+        + custom_fields (array[CustomFieldValue], optional)
+
++ Response 201 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + type: `meeting` (string)
```
Comment (mrtus) [inline src/04-calendar/meetings.apib:257]: It creates an `event` 😉 
```suggestion
            + type: `event` (string)
```

Comment (tiagobastos) [reply src/04-calendar/meetings.apib:257]: The endpoint is `/meetings.schedule` it returns a meeting UUID.

Comment (mrtus) [reply src/04-calendar/meetings.apib:257]: Ahjajajaj its the meeting creation endpoint, nevermind! I was think about `/tasks.schedule` 🤦

**Review by mrtus — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Review by gdegirmenci — DISMISSED** [review]

File: apiary.apib [hunk 3]
```diff
@@ -3228,7 +3233,9 @@ Get a list of meetings.
                     + Members
                         + open
                         + done
-
+                + recurrence (object, nullable)
+                    + type: `recurrence` (string)
```
Comment (mrtus) [inline apiary.apib:3243]: Q: We don't expose this resource anywhere yet, do we?

To which type refers this id then? Without exposing the resource I think we shouldn't include this property yet.

**Review by mrtus — DISMISSED** [review]

File: apiary.apib (hunk 3 above)
Comment (joaoinacio) [reply apiary.apib:3243]: > Q: We don't expose this resource anywhere yet, do we?
> 
> To which type refers this id then? Without exposing the resource I think we shouldn't include this property yet.

🤔 Right. the goal is to be able to list recurring meetings from a recurrence id, so providing the recurrence (definition) isn't necessary for now.
We can omit the type property for now and provide just the id, which is the necessary info to have.

**Review by gdegirmenci — DISMISSED** [review]

File: apiary.apib (hunk 3 above)
Comment (mrtus) [reply apiary.apib:3243]: Okay, where would that id then come from? From existing meetings which return the recurrence?

Though, I wouldn't really remove the type if we'd still expose the id.

**Review by joaoinacio — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Pull request merged**